### ISSUE TITLE
Disable workbench.action.focusSecondEditorGroup making new window

### ISSUE THREAD

**Issue by safield** [Microsoft/vscode#60052]
Current Behavior:

If I have N editor groups (window panes) open, and I hit the keyboard shortcut to execute...

workbench.action.focus[N+1]EditorGroup

It will create a new editor group....which is an undesirable behaviour for me.

Concrete Example:

If I have 2 editor groups open, and I accidentally hit shortcut for workbench.action.focusThirdEditorGroup, it will then create a new third window.

This a massive source of frustration for me. I am commonly jumping between 4 different editor groups rapidly, and a sudden mistype and it creates a new window, and throws a total wrench into my flow and productivity.

Long story short, the command for focusing editor groups should JUST focus editor groups. There should be a setting to disable this behaviour, or just remove it completely.
